README.md

```diff
@@ -15,67 +15,120 @@ Sorting Hat is a part of the [GrimoireLab toolset](https://grimoirelab.github.io
 
 In the context of GrimoireLab, Sorting Hat is usually run after data is retrieved with [Perceval](https://github.com/chaoss/grimmoirelab-perceval), to store the identities obtained into its database, and later merge them into individuals (and maybe affiliate them).
 
+
+## Requirements
+
+* Python >= 3.6
+* Poetry >= 1.1.0
+* MySQL >= 5.7 or MariaDB 10.0
+* Django = 2.1
```

Comment:
Update the Django version to 2.2 since we have updated it recently.
https://github.com/chaoss/grimoirelab-sortinghat/pull/490

Reply:
Done.